Jenkinsfile

```diff
@@ -22,8 +22,10 @@ pipeline {
     }
     post {
         always {
-            sh './tests/copy_files'
-            archiveArtifacts "cilium-files.tar.gz"
+            sh './tests/copy_files || true'
+            archiveArtifacts artifacts: "cilium-files-runtime.tar.gz", allowEmptyArchive: true
+            sh './tests/k8s/copy_files || true'
```

Comment:
I think @tgraf solve this part by writing `exit 0` in the `copy_files` but both me and him saw a PR timing out and jenkins assumed that all tests passed successfully and we weren't sure if it was due `exit 0`. I'm fine keeping this `|| true`